src/BlazorWebView/src/Maui/Windows/WinUIWebViewManager.cs

```diff
@@ -109,7 +109,7 @@ protected override async Task HandleWebResourceRequest(CoreWebView2WebResourceRe
 					{
 						using var contentStream = await Package.Current.InstalledLocation.OpenStreamForReadAsync(relativePath);
 
-						var memStream = new MemoryStream();
+						using var memStream = new MemoryStream();
 						contentStream.CopyTo(memStream);
 						stream = new InMemoryRandomAccessStream();
 						await stream.WriteAsync(memStream.GetWindowsRuntimeBuffer());
```

Comment:
Nit: Since these same 4-ish lines are copied 3 times in the same method, should we consider pulling them into their own method?